Removed forward slashes

bradydowling opened the pull request:
Removed forward slashes causing runner to dysfunction. See https://github.com/c9/core/issues/26#issuecomment-77999985

bradydowling committed:
Removed forward slashes

Removed forward slashes causing runner to dysfunction. See https://github.com/c9/core/issues/26#issuecomment-77999985

lennartcl commented on builders/typescript.build:
```diff
@@ -1,4 +1,4 @@
 {
-    "cmd": ["tsc", "--out", "${file/\\.ts/\\.js/}", "$file"],
+    "cmd": ["tsc", "--out", "${file\\.ts\\.js/}", "$file"],
```
@bradydowling shouldn't the --out parameter be removed as done in c9/core#26 (comment)?

javruben replied:
@bradydowling did you test this?

bradydowling committed:
Modified typescript builder

I've implemented the changes mentioned in the previously referenced ticket and tested them.

bradydowling committed:
Removed extraneous "--out" parameter
